### ISSUE TITLE
Cannot find module 'lodash'

### ISSUE THREAD

**Issue by eduncan911** [mastercactapus/atom-go-quick-import#4]
Thanks for v0.3.0 release for the attempt to fix multiple GOPATHs in #3 !

I upgraded immediately.  But...  I get this on startup now:

```
Cannot find module 'lodash'
Hide Stack Trace
Error: Cannot find module 'lodash'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/eric/sync/home/.atom/packages/go-quick-import/lib/go-quick-import-view.js:21:15)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:109:30)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/eric/sync/home/.atom/packages/go-quick-import/lib/go-quick-import.js:7:26)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:109:30)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:216:21)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:766:27)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:121:28
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:94:15)
    at Package.module.exports.Package.load (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:108:12)
    at PackageManager.module.exports.PackageManager.loadPackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:468:14)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:412:19
    at Config.module.exports.Config.transact (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:312:16)
    at PackageManager.module.exports.PackageManager.loadPackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:407:19)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:811:28
```

**Comment by mastercactapus** [comment]
Seems like babel didn't want to overwrite a file. I'm not seeing this anymore with 0.3.1.

**Comment by eduncan911** [comment]
yep, fixed w/0.3.1!!
